template.qmd

```diff
@@ -3,6 +3,9 @@ title: Quarto Template for Springer Nature
 format:
   nature-pdf:
     journal: "sn-nature"
+    classoption:
+      - lineno
+    cite-method: citeproc
```

Comment:
```suggestion
    cite-method: natbib
```

Reply:
Another option might be to download and vendor csl equivalents of each of the bst files from, for example, [zotero's repository](https://www.zotero.org/styles?q=springer) since pandoc and quarto by extension sort of prefer citeproc. This might be complicated to maintain so it could also just be something to mention in the docs.

Reply:
ff74ecd9e0a6bf58c85840c9d731ac63338405e2 makes this comment irrelevant.
The default should remain citeproc.